backend/app/Http/Controllers/Api/Auth/ResetPasswordController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Http\Controllers\Api\Auth;
+
+
+use App\Http\Requests\ResetPasswordHttpRequest;
+use App\Http\Controllers\Controller;
+use App\Action\Auth\SendResetPasswordLinkAction;
+use App\Action\Auth\ResetPasswordRequest;
+use App\Exceptions\UserByEmailNotFoundException;
+
+class ResetPasswordController extends Controller
+{
+    private $sendLinkAction;
+
+    public function __construct(SendResetPasswordLinkAction $sendLinkAction)
+    {
+        $this->sendLinkAction = $sendLinkAction;
+
+    }
+
+    public function sendPasswordResetLink(ResetPasswordHttpRequest $request)
+    {
+        try {
+            $serviseResponse = $this->sendLinkAction->execute(
+                new ResetPasswordRequest(
+                    $request->get('email')
+                )
+            );
+        } catch (UserByEmailNotFoundException $exception) {
```

Comment:
such as you changed render method in exception handler you do not need try catch here anymore